public/locales/en/translation.json

```diff
@@ -417,6 +417,9 @@
         "releaseDate": "Release Date"
     },
     "library": {
+        "hideAlphabetFilter": "Hide Alphabet Filter",
+        "refresh": "Refresh",
```

Comment:
where is this `"Refresh"` string coming from? is this added automatically when you run `pnpm i18next`?